nodestream/project/project.py

```diff
@@ -118,7 +119,6 @@ def to_file_data(self) -> dict:
                 for scope in self.scopes_by_name.values()
                 if scope.persist
             },
-            "plugins": [plugin.to_file_data() for plugin in self.plugins or []],
```

Comment:
Should we remove this? We don't want to clobber people's `nodestream.yaml` files.

Reply:
I don't think we need this line, but maybe theres something i'm missing